lib/matplotlib/colorbar.py

```diff
@@ -782,7 +781,7 @@ def _add_solids(self, X, Y, C):
                   alpha=self.alpha,
                   edgecolors='None')
         _log.debug('Setting pcolormesh')
-        col = self.ax.pcolormesh(*args, **kw)
+        col = self.ax.pcolormesh(*args, **kw, shading='flat')
```

Comment:
Does this mean that colorbars with 256 colors have been dropping the last color and really only silently pcolormesh-ing 255 colors? Will this start raising deprecation warnings everywhere colorbar is called now?

Reply:
This was some strange interaction with a manually created boundary norm only.  I didn't *fully* track it down, but CI will trip on any warnings, and we don't have any, so I think in general colorbars are unaffected.